lm-agent/tests/parsing/test_flexlm.py

```diff
@@ -44,35 +46,43 @@ def test_parse_feature_line(line, result):
     [
         (
             "    user1 myserver.example.com /dev/tty (v62.2) (myserver.example.com/24200 12507), start Thu 10/29 8:09, 29 licenses",
-            {
-                "user_name": "user1",
-                "lead_host": "myserver.example.com",
-                "booked": 29,
-            },
+            LicenseUsesItem(
+                **{
+                    "username": "user1",
+                    "lead_host": "myserver.example.com",
+                    "booked": 29,
+                }
+            ),
```

Comment:
Why use a dict here instead of just using kwargs?

Reply:
The tests were using the dicts before, so it was faster to update them this way.
But since that may difficult the readabily, I changed it now to kwargs.